app/components/assets/bulk-actions-dropdown.tsx

```diff
@@ -249,6 +260,24 @@ function ConditionalDropdown() {
               </DropdownMenuItem>
             </When>
 
+            <DropdownMenuItem
```

Comment:
Please make this to be the first action in the list, and it should be separated from the one below with a border. Like we usually do and separate them in groups.